lib/fathead/npm_cli/parse.py

```diff
@@ -0,0 +1,216 @@
+        '',               # 5.no categories
+        '',               # 6.leave empty
+        '',               # 7.no related topics
+        '',               # 8.leave empty
+        '',               # 9.an external link back to home
+        '',               # 10.no disambiguation
+        '',               # 11.images
+        '',               # 12.abstract
+        ''                # 13.url to doc
+        ]
+
+    OUTPUL_FILE.write('{}\n'.format('\t'.join(list_of_data)))
+
+    for i in code.split('\n'):
+        # Check explicitly for alias and aliases, both follow diff pattern
+        if i.startswith('aliases'):
+            temp = str(i.split('aliases: ')[-1])
+            temp = temp.split(', ')
+
+            for alias in temp:
+                list_of_data = [
+                    alias,            # 1. article title
+                    'R',              # 2.type is article
+                    new_title,        # 3.no redirect data
+                    '',               # 4.leave empty
+                    '',               # 5.no categories
+                    '',               # 6.leave empty
+                    '',               # 7.no related topics
+                    '',               # 8.leave empty
+                    '',               # 9.an external link back to home
+                    '',               # 10.no disambiguation
+                    '',               # 11.images
+                    '',               # 12.abstract
+                    ''                # 13.url to doc
+                    ]
+
+                OUTPUL_FILE.write('{}\n'.format('\t'.join(list_of_data)))
+
+            continue
+
+        if i.startswith('alias'):
+            command = i.split('npm ')[-1]
+
+            list_of_data = [
+                command,          # 1. article title
+                'R',              # 2.type is article
+                new_title,        # 3.no redirect data
+                '',               # 4.leave empty
+                '',               # 5.no categories
+                '',               # 6.leave empty
+                '',               # 7.no related topics
+                '',               # 8.leave empty
+                '',               # 9.an external link back to home
+                '',               # 10.no disambiguation
+                '',               # 11.images
+                '',               # 12.abstract
+                ''                # 13.url to doc
+            ]
+
+            OUTPUL_FILE.write('{}\n'.format('\t'.join(list_of_data)))
+
+            continue
+
+        # Other irrelevant info
+        if not i or not i.startswith('npm'):
+            continue
+
+        command = re.split(r' [<[()]', i)[0].replace('npm ', '')
+        temp = re.findall(r'[^a-z](-+[a-z]+)', i)
+
+        if not temp and not command == title:
+            command_dict[command] = True
+
+            list_of_data = [
+                command,          # 1. article title
```

Comment:
nit: seeing as you're using this array block in several places, it looks like you might want to abstract this into a `create_redirect` function or similar.